docs/dev/fundamentals/zkSync.md

```diff
@@ -0,0 +1,72 @@
+# Introduction to zkSync
+
+## Perequisite
+
+If you are unfamiliar with rollups, you should cover the [rollups basics](./rollups.md) and read about zk rollups and optimistic rollups, before learning about zkSync.
+
+**zkSync** is a [ZK rollup](./rollups.md) trustless protocol that utilises zero-knowledge proofs to provide scalable low-cost transactions on Ethereum. 
+zkSync ensures that all assets are stored in a single smart contract on the mainchain, while computation and storing data are performed off-chain. As all transactions are proven on the Ethereum mainchain, users enjoy the same security level as in Ethereum.
+
+zkSync 2.0 is made to look and feel like Ethereum, but with lower fees. Just like on Ethereum, smart contracts are written in Solidity/Vyper and can be called using the same clients as the other EVM-compatible chains.
+
+zkSync 2.0 doesn't require you to register a separate private key to use before usage, it supports existing Ethereum wallets which work out of the box.
+
+
+At this time zkSync is solely run and operated by the zkSync team's servers, and is therefore centralized; however, this will be be transitioned to a decentralized system in the near future.
+
+## zkSync in comparison
+
+zkSync [stands out remarkably](https://blog.matter-labs.io/evaluating-ethereum-l2-scaling-solutions-a-comparison-framework-b6b2f410f955) in security and usability among existing L2 scaling solutions. 
+Thanks to the combination of cutting-edge cryptography and on-chain data availability, zkRollup (the core network of zkSync) is the only L2 scaling solution that doesn't require any operational activity to keep the funds safe. 
```

Comment:
> zkRollup (the core network of zkSync)

But we have already described above that zkSync is a zk rollup :)

If you were talking about zkRollup as the antonym for zkPorter from [here](https://blog.matter-labs.io/zkporter-a-breakthrough-in-l2-scaling-ed5e48842fbf), then I think it would require prior introduction to what it is, etc. It is better to talk only about zkSync as whole zkRollup for now